test/e2e/config/docker.yaml

```diff
@@ -49,16 +49,25 @@ providers:
       new: --metrics-addr=:8080
     files:
     - sourcePath: "../data/shared/v1alpha4/metadata.yaml"
-  - name: v1.2.5 # latest published release in the v1beta1 series; this is used for v1beta1 --> main clusterctl upgrades test only.
-    value: "https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.2.5/core-components.yaml"
+  - name: v1.2.7 # supported release in the v1beta1 series; this is used for v1beta1 --> main clusterctl upgrades test only.
```

Comment:
With the new policy change v1.2 and v1.3 are both supported release versions. Therefore I am keeping the v1.2 -> main upgrade test (generally this will be dropped in favor of v1.3).

Thoughts?

Reply:
I'm +1 for having tests for all the supported versions